### PR TITLE
Adds stat checks to all AI malf powers that lack them

### LINKED ISSUE #8853: Malf powers work when dead
I think this may have been reported before, but dead Malf players can still use shit like blackout, after they die.

Only really an issue with mulligan, but fairly significant.

### PR DESCRIPTION
**Pull request by Incoming5643**
As well as one for buying powers in the first place.

Fixes #8853

### CODE REVIEW

**Comment by PsiOmegaDelta** [comment]
Almost seems like all that repeated code could be made into a proc.

**Comment by Incoming5643** [comment]
is there an isdead, I never really bothered to look

**Comment by phil235** [comment]
Yeah, all this copypasta is bad, try to make only one message that says "You cannot usr your powers while you are dead!" or something.

**Comment by Incoming5643** [comment]
ok but I assure you, it's not gonna look MUCH better

**Comment by Miauw62** [comment]
Just use canusetopic
Op 9-apr.-2015 19:28 schreef "Incoming5643" <notifications@github.com>:

**Comment by Incoming5643** [comment]
boy I sure am good at wasting commits

File: code/game/gamemodes/malfunction/Malf_Modules.dm [hunk 1]
```diff
@@ -439,6 +474,10 @@
 		return
 	var/mob/living/silicon/ai/A = usr
 
+	if(A.stat == DEAD)
+		A <<"You are already dead!" //Omae Wa Mou Shindeiru
```
Comment (Cheridan) [inline code/game/gamemodes/malfunction/Malf_Modules.dm]: muda da :tractor:

**Pull request merged**